packages/dashboard/package.json

```diff
@@ -76,6 +76,7 @@
     "redux-saga-test-plan": "^4.0.6",
     "rimraf": "^3.0.2",
     "rollup": "2.76.0",
+    "rollup-plugin-dts": "^5.2.0",
```

Comment:
I removed this package because it broke the build for twinmaker. It's not included in npmpm

Reply:
ah then we need another way to generate the d.ts file - I think we can just use `tsc`

Reply:
removed.